docs/components/customization/user-profile.mdx

```diff
@@ -116,301 +34,321 @@ All props below are required.
-      import { CustomIcon, Icon } from "../icons";
-
-      const UserProfilePage = () => (
-        <UserProfile path="/user-profile" routing="path">
-          <UserProfile.Page label="Custom Page" url="custom" labelIcon={<CustomIcon />}>
-              <MyCustomPageContent />
-          </UserProfile.Page>
-          <UserProfile.Link label="Homepage" url="/home" labelIcon={<Icon />} />
-          <UserProfile.Page label="account" />
-          <UserProfile.Page label="security" />
-        </UserProfile>
-      );
+## Advanced use cases
 
-      export default UserProfilePage;
-      ```
-
-      ```jsx filename="/pages/user-profile/[[...index]].tsx"
-      import { UserProfile } from "@clerk/nextjs";
-      import { MyCustomPageContent } from "../components";
-      import { CustomIcon, Icon } from "../icons";
-
-      const UserProfilePage = () => (
-        <UserProfile path="/user-profile" routing="path">
-          <UserProfile.Page label="Custom Page" url="custom" labelIcon={<CustomIcon />}>
-              <MyCustomPageContent />
-          </UserProfile.Page>
-          <UserProfile.Link label="Homepage" url="/home" labelIcon={<Icon />} />
-          <UserProfile.Page label="account" />
-          <UserProfile.Page label="security" />
-        </UserProfile>
-      );
+### Reorder default routes
 
-      export default UserProfilePage;
-      ```
-    </CodeBlockTabs>
-  </Tab>
+If you want to reorder the default routes (`Account` and `Security`) in the `<UserProfile />` navigation sidebar, you can use the `<UserProfile.Page />` component with the `label` prop set to `'account'` or `'security'`. This will target the existing default page and allow you to rearrange it.
+
+Note that the first item in the navigation sidebar cannot be a `<UserProfile.Link />` or `<UserButton.UserProfileLink />` component.
 
+<Tabs type="usage" items={["Dedicated Page", "<UserButton />"]}>
   <Tab>
-      ```jsx filename="/user-profile.tsx"
-      import { UserProfile } from "@clerk/clerk-react";
-      import { MyCustomPageContent } from "../components";
-      import { CustomIcon, Icon } from "../icons";
-
-      const UserProfilePage = () => (
-        <UserProfile path="/user-profile" routing="path">
-          <UserProfile.Page label="Custom Page" url="custom" labelIcon={<CustomIcon />}>
-              <MyCustomPageContent />
-          </UserProfile.Page>
-          <UserProfile.Link label="Homepage" url="/home" labelIcon={<Icon />} />
-          <UserProfile.Page label="account" />
-          <UserProfile.Page label="security" />
-        </UserProfile>
-      );
+    It's important to note that the first page in the list will always be rendered under the root path that you set with the `path` prop. It's `url` prop will be ignored, so you can pass any value you want. In the following example, `path` is set to `/user-profile`, so the `<CustomPage />` is rendered under the `/user-profile` path.
```

Comment:
```suggestion
<Callout type="info">
The first page in the list will always be rendered under the root path defined with the `path` prop. Its `url` prop will be ignored.
    
In the following example, `path` is set to `/user-profile`, so the `<CustomPage />` is rendered under the `/user-profile` path.
</Callout>
```

May not necessarily need to be a callout, but I think it would look a little better. Also shortened it a bit as it looks a little intimidating in the preview to switch to the Dedicated Page tab and see a whole new paragraph IMO